src/System.Memory/ref/System.Memory.cs

```diff
@@ -184,51 +180,50 @@ public partial struct Enumerator
             public bool MoveNext() { throw null; }
         }
     }
-    public ref partial struct SequenceReader<T> where T : unmanaged, System.IEquatable<T>
+    public static partial class SequenceReaderExtensions
     {
-        public SequenceReader(System.Buffers.ReadOnlySequence<T> buffer) { }
-        public bool IsLastSegment { get { throw null; } }
-        public bool End { get { throw null; } }
-        public ReadOnlySequence<T> Sequence { get { throw null; } }
-        public SequencePosition Position { get { throw null; } }
-        public ReadOnlySpan<T> CurrentSpan { get { throw null; } }
-        public int CurrentSpanIndex { get { throw null; } }
-        public ReadOnlySpan<T> UnreadSpan { get { throw null; } }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out short value) { throw null; }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out int value) { throw null; }
+        public static bool TryReadBigEndian(this ref System.Buffers.SequenceReader<byte> reader, out long value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out short value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out int value) { throw null; }
+        public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out long value) { throw null; }
+        public static bool TryRead<T>(this ref System.Buffers.SequenceReader<byte> reader, out T value) where T : struct { throw null; }
+    }
+    public ref partial struct SequenceReader<T> where T : struct, System.IEquatable<T>
+    {
+        public SequenceReader(System.Buffers.ReadOnlySequence<T> buffer) { throw null; }
```

Comment:
Regenerate based on new sources so that arg names match.

Reply:
This still needs fixing.

Reply:
Whoops, sorry, missed this comment. I'll fix shortly.